chat-sdk-ui/src/main/java/co/chatsdk/ui/chat/ChatActivity.java

```diff
@@ -82,6 +82,7 @@
     // If we are showing a temporary screen like the sticker message screen
     // this should be set to no
     protected boolean removeUserFromChatOnExit = true;
+    protected boolean addUserToChatOnEnter = true;
```

Comment:
I think it would be better if this change were implemented using a subclass and not committed to the master. This code my break in the future if the security rules are changed so you have to be a member of a public chat to send a message... so it's better that it isn't in the main code. You can just create a `ChatActivity` subclass and implement this functionality there.